resolwe/flow/processes/arciver.yml

```diff
@@ -0,0 +1,26 @@
+- slug: arciver
+  name: Archive Files
+  data_name: File archive
+  requirements:
+    expression-engine: jinja
+  version: 0.0.1
+  type: "data:archive:"
+  category: other
+  persistence: TEMP
+  description: Create an archive of input files
+  input:
+    - name: data
+      label: Data list
+      type: list:data
+    - name: fields
+      label: Output file fields
+      type: list:basic:string
+  output:
+    - name: archive
+      label: Archive
+      type: basic:file
+  run:
+    language: bash
+    program: |
+      zip -0 results.zip {% for d in data %}{% for f in fields%}"{{d[f].file}}" {% endfor %}{% endfor %}
+      echo "{\"archive\":{\"file\":\"results.zip\"}}"
```

Comment:
Oh yes, we still have this as a separate package. We should really implement all these functions as predefined macros available as part of the `bash` language, as otherwise we will have these echos everywhere.